mutation/frozen_mutation.hh

```diff
@@ -233,6 +233,9 @@ frozen_mutation freeze(const mutation& m);
 std::vector<frozen_mutation> freeze(const std::vector<mutation>&);
 std::vector<mutation> unfreeze(const std::vector<frozen_mutation>&);
 
+void freeze(bytes_ostream& out, const mutation&);
```

Comment:
Let's call it freeze_into() to make it clearer (I hope).

Reply:
freeze_to(dst, src) maybe? Into sounds like morph.

Reply:
yes

Reply:
@elcallio did you rename the function and this conversation can be resolved?